### ISSUE TITLE
Enhancement request: Make "About Ditto" copy-able

### ISSUE THREAD

**Issue by sjvudp** [sabrogden/Ditto#606]
Well the "About Ditto" is hidden within the Options (instead of the top-level menu), and untypically, you cannot copy the contents of the About window to the clipboard (e.g. to paste the exact version number here).

Wouldn't it make sense support-wise to allow copying the contents of the about window? Maybe also provide a button to do that.

See the example from LibreOffice:
![LO-About](https://github.com/sabrogden/Ditto/assets/6734808/f58358df-c7ef-4142-a551-2e8595e57ada)

**Comment by lheydon** [comment]
Yes, exactly this. Plus I'd suggest adding a shortcut to the 'About' screen as a submenu of the 'Help' option. It's where most people would look for it.
